openmrs-central/prod-openmrs-runtime.properties

```diff
@@ -1,5 +0,0 @@
-connection.url=jdbc:mysql://mysql-central:3306/openmrs_fgh_central?autoReconnect=true&sessionVariables=storage_engine=InnoDB&useUnicode=true&characterEncoding=UTF-8&maxReconnects=15
```

Comment:
Why are you deleting these? I think you should use environment variables for the configurable properties